metadata-ingestion/tests/unit/test_glue_source.py

```diff
@@ -641,3 +684,51 @@ def fake_schema_metadata(entity_urn: str) -> models.SchemaMetadataClass:
         output_path=tmp_path / mce_file,
         golden_path=test_resources_dir / mce_golden_file,
     )
+
+
+@pytest.mark.parametrize(
+    "platform_instance, mce_file, mce_golden_file",
+    [
+        (None, "glue_mces.json", "glue_mces_golden_profiling.json"),
+    ],
+)
+@freeze_time(FROZEN_TIME)
```

Comment:
Not clear to me why this function needs to be parameterised if there is only one combination input.